.travis.yml

```diff
@@ -2,18 +2,16 @@ language: node_js
 node_js: '10'
 
 env:
-  global:
-    - FORCE_COLOR=1
+    global:
+        - FORCE_COLOR=1
 
-cache:
-  directories:
-    - ~/.npm
+cache: yarn
 
 script:
-  - npm run prettier
-  - npm run build
+    - yarn run prettier
+    - yarn run build
 
 deploy:
-  - provider: script
-    skip_cleanup: true
-    script: npm run semantic-release
+    - provider: script
+      skip_cleanup: true
+      script: yarn run semantic-release
```

Comment:
The indentation changes here look wrong (we should use 2 spaces for YAML and JSON) and are likely caused by the recreation of the lockfile updating Prettier but not our prettierrc.